.github/workflows/build-hotfix.yml

```diff
@@ -65,3 +66,20 @@ jobs:
           git commit -m "chore(dependencies): Autobump oss clouddriver version"
           git pull
           git push
+
+      # Adding OSS docker build
+      - name: Login to Docker Hub
+        if: startsWith(github.ref, 'refs/heads/release-') && endsWith(github.ref, '_hotfixes')
+        run: docker login -u ${{ secrets.DOCKER_HUB_USER }} -p '${{ secrets.DOCKER_HUB_PASS }}'
+
+      - name: Build Hotfix Release
+        if: startsWith(github.ref, 'refs/heads/release-') && endsWith(github.ref, '_hotfixes')
+        run: |
+          echo "Building Hotfix Image"
+          docker build -t compile -f Dockerfile.compile . && docker build -t armory/$BASE_PROJECT:${{ steps.build-info.outputs.version }} -f Dockerfile.ubuntu
```

Comment:
Is there a reason to use `Dockerfile.ubuntu` instead of `Dockerfile.slim`? As far as I know Halyard and operator install slim version of services.

Reply:
I don't know which is build be default, and I remember hearing something about a potential issue with slim.  I'll go ahead and change this to slim.